services/gallery.js

```diff
@@ -0,0 +1,55 @@
+class GalleryService {
```

Comment:
We shouldn't use ES6 in client since it's not supported by all browsers. That's the same reason we're using sass polyfills.

Reply:
With that in mind I've checked compatibility issues of EC6 first. Since it's been almost three years now from its release in june '15, it's widely supported now as can be seen on https://kangax.github.io/compat-table/es6/ and thought it was safe to use.

(If we were still worried about compatibility issues after seeing that compatibility table, it's also possible to use transpilers like babel loader with webpack.)

I'm think it's a pitty to loose all the advantages and progresses that ES6 provides for this concerns.

Reply:
If compatibility is not concerning, so we shall see if we need sass compatibility mixins. As I see, the issues are in IE11, Android browser, Samsung browser, firefox < 45 and chrome <49. This means about a 17% ([source](https://caniuse.com/#search=es6%20classes).)

In any case, sytactic suggar `class` is more a disadvantage than a progress, since there aren't real oop classes, but just the classic prototypes js used to have. So, if using the same concept, why calling it in a different and confuse way? Just my view on this.

Reply:
For me it simplify syntax and improves readability of code. As can I use states: ES6 classes are syntactical sugar to provide a much simpler and clearer syntax to create objects and deal with inheritance. ;)